src/mmw/js/src/draw/views.js

```diff
@@ -636,19 +636,31 @@ var DrawAreaView = DrawToolBaseView.extend({
         return {
             id: this.id,
             title: 'Draw area',
-            info: 'Lorem ipsum dolor it amet, consectetur',
+            info: 'Free draw an area or place a square kilometer',
             items: [
                 {
                     id: freeDraw,
                     title: 'Free draw',
-                    info: 'Draw a custom area on which you want to perform water quality analysis.',
+                    info: 'Free draw an area of interest polygon, by clicking ' +
+                          'on map and repeatedly clicking at boundary corners. ' +
```

Comment:
This is what they wrote, but I think it should read `on the map` not just `on map`